rotkehlchen/db/upgrade_manager.py

```diff
@@ -75,6 +77,50 @@
 ]
 
 
+class UpgradeProgressHandler():
```

Comment:
A short docstring on what this class is supposed to be used for.
Also rename to `DBUpgradeProgressHandler`